### PR TITLE
NEXUS-43684: Add env to disable zdu

### PR DESCRIPTION
**Pull request by dibastidas**

### CODE REVIEW

File: nxrm-ha/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ serviceAccount:
     # If using externaldns, role with route53 permissions needed by external-dns
     # eks.amazonaws.com/role-arn: arn:aws:iam::000000000000:role/nxrm-nexus-role
 azure:
-  enabled: false  #set to true to enable azure specific yamls/snippets
```
Comment (mpiggott) [inline nxrm-ha/values.yaml:19]: We should try to avoid the reformatting

File: sample-aws-ha-yamls/aws-ha-statefulset.yaml [hunk 2]
```diff
@@ -103,12 +103,15 @@ spec:
                   key: nexus-admin-password
             - name: NEXUS_SECURITY_RANDOMPASSWORD
               value: "false"
+            - name: CLUSTERED_ZERO_DOWNTIME_ENABLED_ENV
+              value: "false"
```
Comment (mpiggott) [inline sample-aws-ha-yamls/aws-ha-statefulset.yaml:107]: Should this be part of the `values` section?

Comment (mpiggott) [reply sample-aws-ha-yamls/aws-ha-statefulset.yaml:107]: Oh, nm this is an example

File: nxrm-ha/values.yaml (hunk 1 above)
Comment (dibastidas) [reply nxrm-ha/values.yaml]: Removed

**Review by bobotimi — APPROVED** [review]

**Review by anjaliarora06 — APPROVED** [review]
I reviewed using github copilot and looks good to me.

**Review by mpiggott — APPROVED** [review]

File: sample-aws-ha-yamls/aws-ha-statefulset.yaml [hunk 3]
```diff
@@ -103,6 +103,8 @@ spec:
                   key: nexus-admin-password
             - name: NEXUS_SECURITY_RANDOMPASSWORD
               value: "false"
+            - name: CLUSTERED_ZERO_DOWNTIME_ENABLED_ENV
```
Comment (mpiggott) [inline sample-aws-ha-yamls/aws-ha-statefulset.yaml:106]: I just noticed, this shouldn't have an `_ENV` suffix

File: nxrm-ha/tests/statefulset_test.yaml [hunk 4]
```diff
@@ -30,6 +30,22 @@ tests:
...
+    set:
+      statefulset:
+        container:
+          env:
+            zeroDowntimeEnabled: true
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].env[6]
+          value:
+            name: CLUSTERED_ZERO_DOWNTIME_ENABLED_ENV
```
Comment (mpiggott) [inline nxrm-ha/tests/statefulset_test.yaml:44]: Noted below no `_ENV`

File: sample-aws-ha-yamls/aws-ha-statefulset.yaml (hunk 3 above)
Comment (dibastidas) [reply sample-aws-ha-yamls/aws-ha-statefulset.yaml]: Updated names, thanks!

**Pull request merged**